depends/packages/packages.mk

```diff
@@ -0,0 +1,10 @@
+# zip is currently broken. Add it back to packages when it is fixed.
+packages:=openssl pyqt4 twisted
+native_packages := native_pcre native_psutil native_python2 native_rsync native_sip native_swig native_zlib native_zope.interface
+
+qt_native_packages = native_protobuf
```

Comment:
Maybe @theuni can comment, but it seems like part of protobuf is for the build machine and part is for the target machine, so it is split into native and non-native protobuf. I'll have to look at that more, but I think we do need both, because protobuf depends on native_protobuf and we need protobuf.